app/Concerns/PerformsLicenseChecks.php

```diff
@@ -42,17 +42,23 @@
 
 trait PerformsLicenseChecks
 {
+    /**
+     * @param LicenseType|string|array<LicenseType|string> $licenses
+     */
     public function hasLicenses(Authenticatable $authenticatable, LicenseType | string | array $licenses): ?Response
     {
         return $authenticatable->hasLicense($licenses)
-            ? null
-            : Response::deny('You are not licensed to access this resource.');
+          ? null
+          : Response::deny('You are not licensed to access this resource.');
     }
 
+    /**
+     * @param LicenseType|string|array<LicenseType|string> $licenses
+     */
     public function hasAnyLicense(Authenticatable $authenticatable, LicenseType | string | array $licenses): ?Response
     {
         return $authenticatable->hasAnyLicense($licenses)
-            ? null
-            : Response::deny('You are not licensed to access this resource.');
+          ? null
+          : Response::deny('You are not licensed to access this resource.');
```

Comment:
Please revert this and adjust your workspace settings for our projects to ensure you are using 4 spaces not 2.